src/components/CustomAccordion.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { Accordion, Card, Row, Col } from 'react-bootstrap';
+import { useAccordionToggle } from 'react-bootstrap/AccordionToggle';
+import AccordionContext from 'react-bootstrap/AccordionContext';
+
+import Plus from '../assets/plus';
+import Minus from '../assets/minus';
+import '../index.css';
+
+class CustomAccordion extends React.Component {
+  render() {
+    function IconToggleAccordion({ largeHeader, children, eventKey, callback }) {
+      const currentEventKey = useContext(AccordionContext);
+      const toggleIconOnClick = useAccordionToggle(eventKey, () => callback && callback(eventKey));
+      const isCurrentEventKey = currentEventKey === eventKey;
+      const minusIcon = largeHeader ? <Minus large /> : <Minus />;
+      const plusIcon = largeHeader ? <Plus large /> : <Plus />;
+
+      return (
+        <Row className={`accordion-header ${largeHeader ? 'accordion-header-large' : ''}`} onClick={toggleIconOnClick}>
+          <Col xs="auto"> {isCurrentEventKey ? minusIcon : plusIcon} </Col>
```

Comment:
![image](https://user-images.githubusercontent.com/24930524/87859405-ef4add80-c902-11ea-8883-5ec64d2a07a0.png)
the plus and minus icons look a little bit not vertically centered

Reply:
^^ also I think one of the things we talked about last week was having the content padding line up to where the header starts? not sure if that might look too thick though

Reply:
one more lol sorry the padding between the header and the icon is a bit big on smaller screens (caused by the styles that come with Col >:( bootstrap my enemy)

Reply:
ok real last one not sure if we changed our minds on this but font HelveticaNeueBold for the heading instead of med?

Reply:
forgot to respond to these! the icons *should* be centered correctly from when I last checked, and we've decided to use med instead of bold. adjusting the body padding to the header looked too off sadly :(